src/app/shared/enums/multicast-type.ts

```diff
@@ -0,0 +1,4 @@
+export enum MulticastType {
+    // ClassB = 'Class-B',
```

Comment:
Add a comment to this just like you have with the other Class-B code to make it stand out. Format it with something like // Class-B: {comment}